src/handle-data.ts

```diff
@@ -52,8 +52,7 @@ export function getTotalPosition(layout: TableLayout) {
  */
 export function getTotalInfo(layout: TableLayout, colIdx: number, pageColIdx: number, numDims: number) {
   if (colIdx >= numDims) return layout.qHyperCube.qGrandTotalRow[colIdx - numDims]?.qText ?? '';
-  if (pageColIdx === 0) return layout.totals.label;
-  return '';
+  return pageColIdx === 0 ? layout.totals.label : '';
```

Comment:
I might be blind but I see no difference here, the logic looks the same to me 😂

Reply:
You are right my mistake here 😵😅

I tried to shorten this:
```ts
if (pageColIdx === 0) return layout.totals.label ?? ''
```
to this: 
```ts
return pageColIdx === 0 ? layout.totals.label : '';
```

but i forgot the nullish thing
now fixed it

Reply:
maybe this is even more readable ?
```ts
if (pageColIdx === 0) return layout.totals.label ?? ''
```